Api/Src/Models/User.ts

```diff
@@ -0,0 +1,62 @@
+import * as mongoose from 'mongoose';
+// import {Grade} from './Grade' - add grade when create grades model
+
+export enum UserType {
+    Candidate,
+    Participant,
+    Mentor, 
+    Admin,
+};
+
+export enum UserStatus {
+    Active, 
+    Deleted, 
```

Comment:
In endpointsDocumentation task I have talked with Filip about userStatus. We should rename Deleted status to Resigned to clarify what does it mean.

Reply:
Done 👍